src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendRecorder.java

```diff
@@ -51,8 +49,39 @@ public VulnerabilityTrendRecorder(List<ThresholdCondition> conditions, String te
         this.queryBy = queryBy;
     }
 
-    public TeamServerProfile getProfile() {
-        return VulnerabilityTrendHelper.getProfile(teamServerProfileName);
+    protected Object readResolve() {
```

Comment:
@serge17 Can you explain a little bit what's going on here?

Reply:
This method is often used in Jenkins plugins to migrate data from one field to another when the field is deprecated. We added new variables "apps" and "applicationId" to profile, and we populate them in this method. It is triggered automatically by Jenkins.